lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
+    const flashProgress = messages.info.flashProgress(currentFlashState);
+    const title = `${packageJSON.displayName} ${DASH_UNICODE_CHAR} ${flashProgress}`;
+    windowProgress.setTitle(title);
```

Comment:
Good point. Here's my proposal:

- Let's ditch the `windowProgress.setTitle()` function
- Store the initial title in this module at the top of the file (that is, read it from the actual window)
- Make `windowProgress.set()` also set the window title, by making use of the initial title we found out before, and appending the dash plus the percentage
- Make `windowProgress.clear()` set the title back to the initial title we calculated

The reason why I don't want `windowProgress.set()` to calculate the title internally is that the way the title is laid out (and the fact that we keep the application name on the left), is not its responsibility. 

I also want to fix `windowProgress.clear()` to not set the title back to `packageJSON.name`, since we're currently assuming the initial value of the window title, which is actually set from the `index.html` file.